package.json

```diff
@@ -230,19 +216,22 @@
 		"@wordpress/viewport": "^2.8.2",
 		"autoprefixer": "^9.7.3",
 		"babel-eslint": "^10.0.3",
+		"babel-jest": "^24.9.0",
 		"babel-plugin-dev-expression": "^0.2.2",
 		"babel-plugin-ramda": "^2.0.0",
 		"babel-runtime": "^6.26.0",
 		"concurrently": "^5.0.0",
 		"core-js": "3.6.0",
 		"cross-env": "6.0.3",
-		"css-loader": "^3.2.0",
 		"cssnano": "^4.1.10",
 		"deep-freeze": "0.0.1",
 		"del": "git+https://github.com/sindresorhus/del.git",
 		"equivalent-key-map": "^0.2.2",
 		"eslint": "^6.8.0",
 		"eslint-config-prettier": "^6.7.0",
+		"eslint-config-react-app": "^5.1.0",
+		"eslint-loader": "3.0.2",
+		"eslint-plugin-flowtype": "3.13.0",
```

Comment:
Yas :+1:

Reply:
Maybe we don't need `eslint-plugin-flowtype`, but that's another discussion :smiley: